### PR TITLE
Update 02-filedir.md

### PR DESCRIPTION
**Pull request by nblouin**
I understand that there is limited time for this exercise, but giving resources for newbies to further their understanding is important while not adding to the time to deliver the module.

Please delete the text below before submitting your contribution. 

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

### CODE REVIEW

**Comment by ErinBecker** [comment]
This addition seems like a useful resource. LGTM.

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ More specifically, when we type `whoami` the shell:
 > The Shell (Bash) tells you that it cannot find the program `mycommand`
 > because the program you are trying to run does not exist on your computer.
 > We will touch on quite a few commands in the course of this tutorial, but there
-> are actually many more than we can cover here.
+> are actually many more than we can cover here.  For a more complete list of common (and less common, but useful) bash commands please look at the ss64 documentation here: https://ss64.com/bash/
```
Comment (colinmorris) [inline _episodes/02-filedir.md]: Could you add the markdown syntax to turn this into a link? i.e. `[link text](www.example.com)`

Comment (colinmorris) [reply _episodes/02-filedir.md]: Also, for the sake of brevity, what do you think about keeping the text the same, and turning the "many more than we can cover here" text into a link to that ss64 documentation?

**Comment by gcapes** [comment]
Thanks for your contribution - this looks like a good resource. However, I'm closing this as a stale PR. The link would currently fit better in episode 1 if you're interested to submit another PR?

**Pull request closed without merging**